telemetry/definitions/commonDefinitions.json

```diff
@@ -5971,6 +5994,36 @@
                 }
             ]
         },
+        {
+            "name": "languageServer_setup",
+            "description": "LSP setup event",
+            "metadata": [
```

Comment:
This is a passive event because users do not take an actual action for this to occur, please indicate the same. In addition fields of duration and result is missing from this shape

Reply:
> but we don't have support for `result`, so this field needs to be added to the metadata.

Can that be prioritized? Adding `result` explicitly to every metric confuses the messaging for partner teams.